assets/src/edit-story/app/media/media3p/api/test/useMedia3pApi.js

```diff
@@ -77,27 +112,69 @@ describe('useMedia3pApi', () => {
             },
           },
           creationDate: '1234',
-          height: 100,
+          height: 3536,
           id: 'media/unsplash:1234',
           length: null,
           lengthFormatted: null,
           local: false,
-          mimeType: 'image/png',
+          mimeType: 'image/jpeg',
           poster: null,
           posterId: null,
           sizes: {
+            web_stories_thumbnail: {
+              file: 'media/unsplash:1234',
+              source_url:
+                'https://images.unsplash.com/photo-1515516089376-88db1e26e9c0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEzNzE5M30&fm=jpg&w=340&h=227&fit=max',
+              mime_type: 'image/jpeg',
+              width: 340,
+              height: 227,
+            },
+            '1060_707': {
```

Comment:
1060_707, 2121_1414, 3182_2121 sounds a bit random. What are they used for specifically?

Reply:
The API returns a minimum of 340 (which is the width of the media library), and then increments of 20%, 40%, 60%, 80% and 100% of the image's full size. The objective is to allow us to use srcsets properly, by using the images that make sense depending on where they will be displayed and the device's dpr. Today, the web_stories_thumbnail width of 150px results in blurry pictures when the mediia gallery is displayed in rows (eg: when the image width spans the entire gallery), and also for devices with dpr > 1 (eg: a Macbook Pro). To be honest, I'd like to not use the keys of the "sizes" object at all, and simply choose the URL that is best suited for each situation. So the getThumbnailUrl.js file would change to instead return the appropriate url for the given width (in the case of the media gallery, the width of it). WDYT?